src/format.rs

```diff
@@ -791,6 +791,7 @@ mod test {
     use super::EncryptionFormat;
 
     #[test]
+    #[allow(clippy::eq_op)]
```

Comment:
Please reduce the ```allow```'s scope to just the assert_eq!(), i.e., don't have it cover the whole test.

Reply:
Unfortunately, if I do that, it won't allow the `clippy::eq_op` lint.  It will show the same "identical args used" error as would appear if the "allow" statement wasn't there.

(I believe we ran into this exact issue last time I added an allow to a test or function.)

Reply:
That is really very weird...I don't see any reason why the assert_ne! statement should trigger the lint.

Reply:
@bgurney-rh Do this "trick": https://github.com/stratis-storage/libcryptsetup-rs/pull/116/files#diff-c875d3cb38acba1839738726b3d4aa920bec536459b52d5bc5069a91dd1620f5R795. It's _very_ wierd, unless you make the inner scope with the extra braces, the macro expansion makes the allow directive disappear.

Reply:
Done.